config/environments/test.rb

```diff
@@ -21,6 +22,7 @@
   # Show full error reports and disable caching.
   config.consider_all_requests_local       = true
   config.action_controller.perform_caching = false
+  config.cache_store = :null_store
```

Comment:
I'm curious if this will affect tests that rely on cache (for example, tests for the cached fields)... but anyway, if tests pass, this is good to go!